src/geo/cartodb-layer-group-view-base.js

```diff
@@ -65,7 +65,7 @@ CartoDBLayerGroupViewBase.prototype = {
         tilejson: '2.0.0',
         scheme: 'xyz',
         grids: this.model.getGridURLTemplates(layerIndexInLayerGroup),
-        tiles: this.model.getTileURLTemplates(),
+        tiles: [this.model.getTileURLTemplate()],
```

Comment:
Ok. From what I understand `this.model.getTileURLTemplate()` will return the template with the `{s}` placeholder, right? Wouldn't we need to generate different URLs for each subdomain?